proxy/src/main/java/org/apache/rocketmq/proxy/remoting/pipeline/AuthenticationPipeline.java

```diff
@@ -36,13 +33,9 @@ public AuthenticationPipeline(List<AccessValidator> accessValidatorList) {
 
     @Override
     public void execute(ChannelHandlerContext ctx, RemotingCommand request, ProxyContext context) throws Exception {
-        ProxyConfig config = ConfigurationManager.getProxyConfig();
-        if (config.isEnableACL()) {
```

Comment:
I think should keep  ```if (config.isEnableACL()) ``` in case the config is dynamically modified in the future

Reply:
you mean EnableACL is true at first, but changed when executing?

Reply:
Yes, now  broker config can be changed dynamically, and proxy config may be in the future

Reply:
ok